src/openms/source/ANALYSIS/QUANTITATION/AbsoluteQuantitation.cpp

```diff
@@ -512,9 +514,10 @@ namespace OpenMS
       }
       else
       {
-        break;
+        return optimal_calibration_found;
       }
     }
+    return optimal_calibration_found;
```

Comment:
return false; // no optimal calibration found

hope I got it right but it seemed to me that the variable is not needed and thus can be removed